lib/mayaHydra/hydraExtensions/adapters/directionalLightAdapter.cpp

```diff
@@ -53,11 +53,14 @@ class MayaHydraDirectionalLightAdapter : public MayaHydraLightAdapter
 
     void _CalculateLightParams(GlfSimpleLight& light) override
     {
+        //To simulate a directional light which has no actual position, but doesn't seem to be supported in hydra, we set a position very very far
+        //so it looks like a directional light.
         // Directional lights point toward -Z, but we need the opposite
         // for the position so the light acts as a directional light.
-        const auto direction = GfVec4f(0.0, 0.0, 1.0, 0.0) * GetTransform();
+        static const double farfarAway {1.0e15};//we use a point on the Z axis far far away
```

Comment:
looks better to define a shared variable for it, as it's used elsewhere?

Reply:
Right, but both are not in the same project so I thought it would be simpler to do so.

Reply:
As discussed offline, I'll try to create a shared function that deal with that light direction computation.

Reply:
I created a shared function and added some tests.